site/src/api/typesGenerated.ts

```diff
@@ -1908,3 +1908,92 @@ export const WorkspaceTransitions: WorkspaceTransition[] = [
 
 // From codersdk/workspaceproxy.go
 export type RegionTypes = Region | WorkspaceProxy
+
+// Code generated by 'make site/src/api/typesGenerated.ts'. DO NOT EDIT.
```

Comment:
We should probably adjust this prelude. Since we just concatenating files, we should make this a divider. And this `DO NOT EDIT` just once at the top.